### DISCUSSION TITLE
Consuming SQL Functions with Entity Framework

### DISCUSSION THREAD

**Discussion by giscus** [timdeschryver/timdeschryver.dev#365] [Announcements]
# Consuming SQL Functions with Entity Framework

Using DbFunctions is a method to use database-specific functions in your Entity Framework queries. They allow you to take advantage of features that are specific to a particular database provider, such as SQL Server and use them in your Entity Framework queries in a provider-agnostic way. As an example, let's use the SOUNDEX SQL function.

https://timdeschryver.dev/blog/consuming-sql-functions-with-entity-framework

**Comment by JNSM1** [comment]
Can you return an Entity from a Custom Sql Function?
